pandas/core/frame.py

```diff
@@ -4969,6 +4969,39 @@ def f(x):
 
         return Series(result, index=labels)
 
+    def nunique(self, axis=0, dropna=True):
+        """
+        Return Series with number of distinct observations over requested
+        axis.
+
+        .. versionadded:: 0.20.0
+
+        Parameters
+        ----------
+        axis : {0 or 'index', 1 or 'columns'}, default 0
+            0 or 'index' for row-wise, 1 or 'columns' for column-wise
```

Comment:
Note that it is not fully consistent throughout frame.py as well. There are also some methods that explain this differently (eg `apply`, `mode`. `corrwith` actually switches the row and column-wise ("0 or 'index' to compute column-wise, 1 or 'columns' for row-wise")). 

The thing is also that the explanation can be different depending on what the function does I think.